backend/src/tasks/helpers/simple-wappalyzer.ts

```diff
@@ -91,7 +96,15 @@ const outOfTheBoxTechnologies = {
   ...z
 };
 
-const parseCookie = (str) => Cookie!.parse(str)!.toJSON();
+const parseCookie = (str) => {
```

Comment:
I think we could probably replace this with `const parseCookie = (str) => Cookie?.parse(str)?.toJSON();` which I think would do the same checks

Reply:
It is partially the same but the change checks for undefined str. Not sure what the behavior of parse(undefined) is.
If you don't mind, I would like to keep it as is.

Reply:
Sounds good

Reply:
How about `Cookie?.parse(str || "")?.toJSON();` -- might just be better for conciseness / so we have less lines of code to maintain, but ultimately your call as to what style you prefer!